tests/src/Api/DeleteApiTest.php

```diff
@@ -28,15 +27,15 @@ class DeleteApiTest extends TestCase
      */
     public function testDeleteDocument(array $clientConfig, array $responses, $expectedResult): void
     {
-        $actualResult = $this->getTestingClient($clientConfig, $responses, [$this, 'inspectRequest'])
+        $actualResult = $this->getTestingClient($clientConfig, $responses, [$this, 'assertRequest'])
             ->deleteDocument('foo');
         $this->assertSame($expectedResult, $actualResult);
     }
 
     /**
      * @param RequestInterface $request
      */
-    public function inspectRequest(RequestInterface $request): void
+    public function assertRequest(RequestInterface $request): void
```

Comment:
revert this change for now, it will be fixed in the middleware changes but for now lets be consistent